.travis.yml

```diff
@@ -109,7 +109,9 @@ jobs:
       before_script:
         - gcloud docker --authorize-only
         - bin/docker-pull-deps
-        - bin/docker-pull master
+         # Pulling master helps with docker build cache, but may fail if we're
+         # adding a new image to the mix.
+        - bin/docker-pull master || true
```

Comment:
TIOLI -- might be good to print that the pull failed to give us some indication that this is happening.

```
- bin/docker-pull master || echo "docker pull failed" >&2
```